### ISSUE TITLE
process_mimic_modified wrongly converts ICD9 codes due to extra double-quote in MIMIC-III

### ISSUE THREAD

**Issue by yenlow** [Optum/retain-keras#17]
Noticed that in MIMIC-III, DIAGNOSIS_ICD.csv has ICD9 codes that are double-quoted, e.g.
```
"ROW_ID","SUBJECT_ID","HADM_ID","SEQ_NUM","ICD9_CODE"
1297,109,172335,1,"40301"
1298,109,172335,2,"486"
1299,109,172335,3,"58281"
```
This is unlike in MIMIC-III demo that is cleaned of all quotes.

The double-quote, if not removed, becomes an extra character when converting to ICD9 quotes, e.g. `5569`
is read as `"5569` and parsed into `"55.69` which is an invalid ICD9 diagnosis code. Without the double-quote, `5569` will be correctly parsed as `556.9`. 

As a workaround, I used the following regex instead of `strip()` to remove undesirable characters including double-quotes (ref: [original code](https://github.com/Optum/retain-keras/blob/4eb41d44e566855eb0e5a0b1df663ce69ef7fdc9/process_mimic_modified.py#L84))
```
tokens = re.sub('"|\s|\n','',line).split(',')
```

**Comment by tRosenflanz** [comment]
Thanks @yenlow . Someone from Optum team should fix this when there is time

**Comment by jstremme** [comment]
@yenlow, great catch.  As @tRosenflanz said, I or someone from my team at Optum will update this in the upcoming weeks.  We have an internal version of this code that has been updated significantly, and we aim to add to these updates to the public repo.

**Comment by jstremme** [comment]
Fixed by PR #28
